src/main/java/org/commcare/formplayer/beans/menus/EntityListResponse.java

```diff
@@ -267,6 +270,10 @@ public void setSortIndices(int[] sortIndices) {
         this.sortIndices = sortIndices;
     }
 
+    public boolean isHasDetails() {
```

Comment:
super nit: awkward name, `doesHaveDetails`?

Reply:
It does need to be `is$booleanVar` for the Java model to Json conversion as a default conversion mechanism. We can still change it but think it's quite normal in Java to have booleans named like this.